### PR TITLE
Add drop-down for coordinate search results.

### PR DESCRIPTION
**Pull request by marionb**
Fixes https://github.com/camptocamp/c2cgeoportal/issues/2294

Examples: 
- Desktop: https://marionb.github.io/ngeo/coordinate_search_result/examples/contribs/gmf/apps/desktop/
- Mobile: https://marionb.github.io/ngeo/coordinate_search_result/examples/contribs/gmf/apps/mobile/

### CODE REVIEW

File: contribs/gmf/src/directives/search.js [hunk 1]
```diff
@@ -347,7 +347,20 @@ gmf.SearchController = function($scope, $compile, $timeout, gettextCatalog,
...
   // For searching coordinates
   this.datasets.push({
-    source: this.createSearchCoordinates_(this.map_.getView())
+    source: this.createSearchCoordinates_(this.map_.getView()),
+    name: 'coordinates',
+    display: 'label',
+    templates: {
+      header: '<div class="search-header">' + gettextCatalog.getString('Recenter to') + '</div>',
+      suggestion: function(suggestion) {
+        var coordiantes = suggestion['label'];
```
Comment (ger-benjamin) [inline contribs/gmf/src/directives/search.js]: typo (coordiantes)

File: contribs/gmf/src/directives/search.js [hunk 2]
```diff
@@ -568,13 +581,14 @@ gmf.SearchController.prototype.getBloodhoudRemoteOptions_ = function() {
 
 /**
  * @param {ol.View} view todo
- * @return {function(string)} todo
+ * @return {function(string, function(Object))} todo
```
Comment (ger-benjamin) [inline contribs/gmf/src/directives/search.js]: Can you change the "todo" (even if that's out of the scope this PR)

**Comment by ger-benjamin** [comment]
All right for me. LGTM

**Comment by fredj** [comment]
please merge, thanks

**Pull request merged**